tensorflow/core/kernels/mkl_pooling_ops_common.cc

```diff
@@ -217,6 +218,7 @@ void MklPoolingBwdPrimitive<T>::Execute(const T* diff_dst_data,
 }
 
 template class MklPoolingBwdPrimitive<float>;
+template class MklPoolingBwdPrimitive<bfloat16>;
 
 #endif
```

Comment:
```suggestion
#endif  // !INTEL_MKL_MKL_ONLY
```

Reply:
thanks for review @penpornk. Pls take a look at my change.